sw/device/tests/uart_tx_rx_test.c

```diff
@@ -67,7 +67,7 @@ typedef enum uart_direction {
  * placed in .data section in the main SRAM. We cannot backdoor write anything
  * in SRAM at the start of the test because the CRT init code wipes it to 0s.
  */
-static volatile const uint8_t kUartIdx = 0x0;
+static volatile const uint8_t kUartIdx = UART_IDX;
```

Comment:
The documentation above only explain the DV use case, can you add a paragraph explaining the sival strategy?

Reply:
Done